steps.md

```diff
@@ -0,0 +1,27 @@
+Workshop steps
```

Comment:
These look identical to those in README.

Reply:
Oh woops, yes I wrote it in `steps.md` then copied it into readme, will remove `steps.md`, my bad.